<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LotteryPublishDialog</class>
 <widget class="QDialog" name="LotteryPublishDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbPrice">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>单价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePrice">
       <property name="placeholderText">
        <string>2.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbTotal">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="text">
        <string>奖池累计</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTotal">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnConfirm">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
